advanced-creation.md

```diff
@@ -111,3 +111,112 @@ const unicorn = got.create(settings);
 // Same as:
 const unicorn = got.extend({headers: {unicorn: 'rainbow'}});
 ```
+
+### Merging instances
+
+#### got.mergeInstances(instanceA, instanceB, ..., [methods])
+
+Merges many instances into a single one:
+- options are merged using [`got.mergeOptions()`](readme.md#gotmergeoptionsparentoptions-newoptions) (+ hooks are merged too),
+- handlers are stored in an array.
+
+##### [instances](readme.md#instances)
+
+##### [[methods]](#methods)
+
+Default: `instances[0].defaults.methods`
```

Comment:
What's the use-case for this?

Reply:
It's not needed at all (that was an old idea). Hang on, I'll fix it.

Reply:
Removed.